docs/tutorials/projects/filtered-camera-module.md

```diff
@@ -0,0 +1,400 @@
+1. Give the service a name, like `my-data-management`, then click **Create**.
+1. On the panel that appears, you can manage the capturing and syncing functions individually.
+   By default, the data management service captures data to the <file>~/.viam/capture</file> directory, and syncs captured data files to the Viam app every 6 seconds (`0.1` minutes in the configuration).
+   Leave the default settings as they are, and click **Save Config** at the bottom of the screen to save your changes.
+
+   {{< imgproc src="/tutorials/filtered-camera-module/configure-data-management.png" alt="The data management service configuration pane with default settings shown for both capturing and syncing" resize="600x" >}}
+
+For more information, see [Add the data management service](/data/capture/#add-the-data-management-service).
+
+### Configure data capture for a camera component
+
+Once you have added the data management service, enable image data capture for your camera component:
+
+1. On your robot's **Config** page in the [Viam app](https://app.viam.com), navigate to the **Components** tab.
+1. In the configuration pane for your configured camera component (named `my-webcam` in the above steps), find the **Data capture configuration** section, and click the **Add method** button to enable data capture for this camera.
+
+   - Set the **Type** to `ReadImage` and the **Frequency** to `0.333`.
+     This will capture an image from the camera roughly once every 3 seconds.
+     You can adjust the capture frequency if you want the camera to capture more or less image data, but avoid configuring data capture to higher rates than your hardware can handle, as this could lead to performance degradation.
+
+   - Set the **Mime type** to `image/jpeg`.
+   - Ensure that the toggle on the right-hand side is set to **On**.
+
+     {{< imgproc src="/tutorials/filtered-camera-module/configure-webcam-data-capture.png" alt="The camera component configuration pane in the Viam app with data capture configured and enabled" resize="1400x" >}}
+
+1. Click **Save Config** at the bottom of the window to save your changes.
+1. In the [Viam app](https://app.viam.com), navigate to the [**Data**](/data/view/) tab, where you should see images captured by your camera component appearing steadily.
+   If you see images appear here, proceed to the next step.
+   If you do not see images appear after a short time, see the [troubleshooting](#troubleshooting) section for further guidance.
+
+For more information see [Configure data capture](/data/capture/#configure-data-capture-for-individual-components) and [Configure cloud sync](/data/cloud-sync/).
+
+## Label images and train a machine learning (ML) model
+
+Now that your camera component is steadily capturing images to the Viam app, you can train a machine learning (ML) model on those images to be able to detect certain objects in the camera feed.
+
```

Comment:
I think this is where you'd want to also say this. 
```suggestion

{{< alert title="Note" color="note" >}}
If you want to skip training your own model, you can use skip to [Upload an existing ML model]() and use a provided model or an exisitng model you already have.
{{< /alert >}}

```

Reply:
Beat you to it :D